tests/all/test_main.py

```diff
@@ -1,139 +1,141 @@
-        self.assertEquals(parsed.mode, ES_MDA_MODE)
-        self.assertEquals(
-            parsed.config, "test-data/local/poly_example/poly.ert")        
-        self.assertEquals(parsed.weights, "4, 2, 1")
-        self.assertEquals(parsed.func.__name__, "run_cli")
-        self.assertFalse(parsed.verbose)
-
-    def test_argparse_exec_workflow(self):
-        parser = ArgumentParser(prog="test_main")
-        parsed = ert_parser(
-            parser, [WORKFLOW_MODE, "--verbose", "workflow_name", 'test-data/local/poly_example/poly.ert'])
-        self.assertEquals(parsed.mode, WORKFLOW_MODE)
-        self.assertEquals(parsed.name, "workflow_name")
-        self.assertEquals(
-            parsed.config, "test-data/local/poly_example/poly.ert")
-        self.assertEquals(parsed.func.__name__, "run_cli")
-        self.assertTrue(parsed.verbose)
-
-    def test_argparse_exec_ensemble_experiment_current_case(self):
-        parser = ArgumentParser(prog="test_main")
-        parsed = ert_parser(parser, [ENSEMBLE_EXPERIMENT_MODE, "--current-case", 'test_case',
-                                     'test-data/local/poly_example/poly.ert'])
-        self.assertEquals(parsed.mode, ENSEMBLE_EXPERIMENT_MODE)
-        self.assertEquals(
-            parsed.config, "test-data/local/poly_example/poly.ert")
-        self.assertEquals(parsed.current_case, 'test_case')
-        self.assertEquals(parsed.func.__name__, "run_cli")
-        self.assertFalse(parsed.verbose)
-    
-    def test_argparse_exec_ensemble_smoother_current_case(self):
-        parser = ArgumentParser(prog="test_main")
-        parsed = ert_parser(parser, [ENSEMBLE_SMOOTHER_MODE, "--current-case", 'test_case',
-                                     "--target-case", 'test_case_smoother',
-                                     'test-data/local/poly_example/poly.ert'])
-        self.assertEquals(parsed.mode, ENSEMBLE_SMOOTHER_MODE)
-        self.assertEquals(
-            parsed.config, "test-data/local/poly_example/poly.ert")
-        self.assertEquals(parsed.current_case, 'test_case')
-        self.assertEquals(parsed.func.__name__, "run_cli")
-        self.assertFalse(parsed.verbose)
-    
-    def test_argparse_exec_ensemble_es_mda_current_case(self):
-        parser = ArgumentParser(prog="test_main")
-        parsed = ert_parser(parser, [ES_MDA_MODE, "--current-case", 'test_case',
-                                     'test-data/local/poly_example/poly.ert'])
-        self.assertEquals(parsed.mode, ES_MDA_MODE)
-        self.assertEquals(
-            parsed.config, "test-data/local/poly_example/poly.ert")
-        self.assertEquals(parsed.current_case, 'test_case')
-        self.assertEquals(parsed.func.__name__, "run_cli")
-        self.assertFalse(parsed.verbose)
-
-
-if __name__ == '__main__':
-    unittest.main()
+if sys.version_info >= (3, 3):
+    from unittest.mock import Mock
+else:
+    from mock import Mock
```

Comment:
Do we need `unittest.mock`'s version of `Mock`? Having 4 lines for an import is fine if it happens once, but I can imagine we'd want to use `Mock` in other tests later on, where it'd be easier to use `from mock import Mock` and later change to `from unittest.mock import Mock` when we drop all pythons <3.3.

Reply:
Will become an issue of semantics, but doing it this way means we only have to delete everything but `from unittest.mock import Mock` when the day comes, won't have to add anything.